app/models/miq_report/generator.rb

```diff
@@ -319,7 +319,7 @@ def generate_basic_results(options = {})
 
     ## add in virtual attributes that can be calculated from sql
     rbac_opts[:extra_cols] = va_sql_cols unless va_sql_cols.blank?
-    rbac_opts[:use_sql_view] = true if db_options && db_options[:use_sql_view]
+    rbac_opts[:use_sql_view] = true unless db_options && db_options[:use_sql_view] == false
```

Comment:
Two alternative ideas on how to handle this:

### Option 1

Maybe make this "falsey", where we still do an "existence" check, but then simply check that the value is falsey:

```ruby
rbac_opts[:use_sql_view] = true unless db_options.has_key?(:use_sql_view) && !db_options[:use_sql_view]
```

### Option 2

Alternatively, we can change this in `Rbac` so that this code here only sets it if it exists:

```ruby
rbac_opts[:use_sql_view] = db_options[:use_sql_view] if db_options.has_key?(:use_sql_view)
```

And then in `Rbac.search`, we do a:

```ruby
def search(options={})
  # ...
  options[:use_sql_view] ||= true

  # ...
end
```

Or do you think that is too heavy handed?

Reply:
Or something like this:

```ruby
rbac_opts[:use_sql_view] = db_options[:use_sql_view] != false
```

I do think there is some merit to considering option 2 though.

You can't memoize with a false value though, so perhaps something like:

```ruby
def search(options={})
  # ...
  options[:use_sql_view] = true unless options[:use_sql_view] = false

  # ...
end
```